src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,19 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map((movie) => (
+      <MovieCard
+        movie={movie}
+        key={movie.imdbId}
+      />
+    ))}
+  </div>
```

Comment:
You had 2 comments about semantic tag ul. Why didn't you fix it?

Reply:
still not fixed